test/zzz_od/application/charge_plan/charge_plan_app/test_check_battery_charge.py

```
@@ -0,0 +1,44 @@
+import pytest
+from test.conftest import TestContext
+
+from zzz_od.application.charge_plan.charge_plan_app import ChargePlanApp
+
+
+class TestCheckBatteryCharge:
+
+    @pytest.mark.parametrize(
+        ('image_name', 'expected'),
+        [
+            # 电量固定为3位数，储蓄电量上限为4位数（2400），以太电池上限为3位数（300）
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)